resources/js/tall-stack/dynamic/reorderSiteAwards/handleResetOrder.ts

```diff
@@ -0,0 +1,52 @@
+import { collectMappedTableRows } from './collectMappedTableRows';
+import type { MappedTableRow } from './index';
+import { computeDisplayOrderValues } from './index';
+
+declare global {
+  interface Window {
+    postAllAwardsDisplayOrder: (awards: Partial<MappedTableRow>[]) => void;
+  }
+}
+
+export function handleResetOrder(): void {
+  if (!confirm('This will reset the order of all awards. Are you sure?')) {
+    return;
+  }
+
+  const rows = Array.from(document.querySelectorAll<HTMLTableRowElement>('.award-table-row'));
+
+  // Sort rows by date of aquisition (ascending)
+  sortAwardsByAwardDate(rows);
+
+  for (const row of rows) {
+    const awardKind = row.getAttribute('data-award-kind');
+    document
+      .querySelector<HTMLTableElement>(`#${awardKind}-reorder-table`)
+      ?.querySelector('tbody')
+      ?.appendChild(row);
+  }
+
+  const mappedTableRows = collectMappedTableRows();
```

Comment:
Performance: `collectMappedTableRows` requeries all rows.

For most users this is not really a big deal. However, some users have 1,000+ rows rendered to the page.

I believe `rows` is virtually identical to `collectMappedTableRows::awardTableRowEls`. Can it be passed as an arg?